### PR TITLE
feat: add local text

### PR DESCRIPTION
**Pull request by cachho**
also related to #37, this PR adds support for any local text that you want to add to the database.

I set the chunk size at 1000, comparable to a PDF. But thinking about it, I guess PDFs are somewhere between chapters and books, and texts tend to be the length of a paragraph or multiple paragraphs. So I'm open to change that, as I was for #31.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,15 @@ To add any web page, use the data_type as `web_page`. Eg:
 app.add('web_page', 'a_valid_web_page_url')
 ```
 
+### Text
+
+To supply your own text, use the data_type as `text` and enter a string. The text is not processed, this can be very versatile. Eg:
+
+```python
```
Comment (taranjeet) [inline README.md:123]: Can you update the example here to
```
app.add_local('text', 'Seek wealth, not money or status. Wealth is having assets that earn while you sleep. Money is how we transfer time and wealth. Status is your place in the social hierarchy.')
```

Want the entire example to be structured around Naval.

File: embedchain/chunkers/text.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from embedchain.chunkers.base_chunker import BaseChunker
+
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+
+
+TEXT_SPLITTER_CHUNK_PARAMS = {
+    "chunk_size": 1000,
```
Comment (taranjeet) [inline embedchain/chunkers/text.py:7]: Hey, can we keep a chunk size of 300 here?

**Review by taranjeet — COMMENTED** [review]
Two minor comments. Rest look goods.

**Comment by cachho** [comment]
Updated with the requested changes.

**Comment by taranjeet** [comment]
Thank you @cachho for the contribution. Loved it.

**Pull request merged**